scripts/sct_maths.py

```diff
@@ -336,10 +352,12 @@ def main(args=None):
         data_out = smooth(data, sigmas)
 
     elif arguments.dilate is not None:
-        data_out = dilate(data, convert_list_str(arguments.dilate, "int"))
+        data_out = sctmath.morphomath(data, filter='dilation', size=arguments.dilate, shape=arguments.shape,
```

Comment:
i tried this way of importing sct to prevent confusion with python's math package, but ran into this issue:
```python
import spinalcordtoolbox as sct
sct.math
Traceback (most recent call last):
  File "/Applications/PyCharm CE.app/Contents/helpers/pydev/_pydevd_bundle/pydevd_exec2.py", line 3, in Exec
    exec(exp, global_vars, local_vars)
  File "<input>", line 1, in <module>
AttributeError: module 'spinalcordtoolbox' has no attribute 'math'
```
any idea how we could do it?

Reply:
Oops. You need to do:

```
import spinalcordtoolbox as sct
import spinalcordtoolbox.math

sct.math
```

Python submodules don't get loaded until they're imported explicitly.

The usual way to handle this is to pre-import all the important modules in `__init__.py`; so put into `spinalcordtoolbox/__init__.py`:

```
import spinalcordtoolbox.math
```

Reply:
ok fixed here d0c942320a2ce786c015ad2746689f95269cab90